### ISSUE TITLE
New linter: detect invalid syntax for markers

### ISSUE THREAD

**Issue by kannon92** [kubernetes-sigs/kube-api-linter#157]
I've seen a few cases where someone adds tags but they forget a space between the comment.

```yaml
//+listType=atomic
//+kubebuilder:validation:MaxItems=16
```

In this case, I think that the tags don't actually get applied. 

Is this worth adding a linter more?

I've found this out when trying to apply other rules and seeing that they didn't actually apply.

**Comment by JoelSpeed** [comment]
Currently, to my knowledge at least, neither controller-gen nor the other K8s related code generators from markers allow `//+` and require `// +`. This is definitely one of the things I had hoped #23 would pick up on

**Comment by everettraven** [comment]
> Currently, to my knowledge at least, neither controller-gen nor the other K8s related code generators from markers allow `//+` and require `// +`. This is definitely one of the things I had hoped [#23](https://github.com/kubernetes-sigs/kube-api-linter/issues/23) would pick up on

If this is something that is not allowed by controller-gen or core kube generators, then it seems reasonable to include this in the scope of #23
